go.mod

```diff
@@ -229,6 +229,7 @@ replace (
 	github.com/dustin/go-humanize => github.com/dustin/go-humanize v1.0.0
 	github.com/elazarl/goproxy => github.com/elazarl/goproxy v0.0.0-20180725130230-947c36da3153 // 947c36da3153 is the SHA for git tag v1.11
 	github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.9.5+incompatible
+	github.com/emicklei/go-restful/v3 => github.com/emicklei/go-restful/v3 v3.7.3
```

Comment:
I've left the `github.com/emicklei/go-restful` (non-go modules) replace in place; I wasn't sure what's common in this scenario, but that replace should (if I'm correct) still help with getting the indirect dependencies to the same / consistent version.

(@dims ?)